metricbeat/module/docker/container/data.go

```diff
@@ -31,7 +32,26 @@ func eventMapping(cont *dc.APIContainers) common.MapStr {
 		"status": cont.Status,
 	}
 
+// Check id container have health metrics configured
+if strings.Contains(cont.Status, "(") && strings.Contains(cont.Status, ")") {
```

Comment:
Ok, that means the data fetching will also work with older version as the state is optional also for >= 1.12.